docs/docs-beta/docs/guides/data-modeling/configuring-assets.md

```diff
@@ -1,5 +1,40 @@
 ---
-title: Configuring assets and ops
-sidebar_label: Configuring assets
+title: Configuring assets and ops via the UI
+sidebar_label: Configuring assets via the UI
 sidebar_position: 50
----
\ No newline at end of file
+---
+
+You will commonly want to manually materialize assets using the Dagster UI to backfill historical data, debug a production issue, or some other one-off task.
+
+Often, you will want to be able to tweak some parameters when materializing these assets. This can be accomplished through the asset configuration system.
+
+## What you'll learn
+
+- How to add configuration to your assets
```

Comment:
This makes me think the section is about configuring the assets, when, if I understand correctly, it's more about "How to define configuration parameters for your assets that can be set when launching a run that targets them".

That's a bit of a mouthful though. Maybe "How to add run configuration schema to your assets"?

Reply:
I agree with Sandy, but I think 'run configuration schema' is more about the 'machinery' than the goal.
Maybe something like 'Configuring Assets when Launched'?

Reply:
"Configuring assets when launching runs" or "a run"?